src/css/responsive.css

```
@@ -0,0 +1,125 @@
+    padding: 6px 8px;
+    font-size: 0.65rem;
+  }
+
+  .footer p {
+    max-width: 95%;
+  }
+
+  .loading {
+    font-size: 1rem;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
```

Stylelint (17.3.0)

[error] Unexpected deprecated property "-webkit-box-direction" (property-no-deprecated)

(property-no-deprecated)